.changes/unreleased/Features-20220408-165459.yaml

```
@@ -0,0 +1,8 @@
+kind: Features
+body: Added favor-state flag to optionally favor state nodes even if unselected node
+  exists
+time: 2022-04-08T16:54:59.696564+01:00
+custom:
+  Author: daniel-murray & josephberni
```

just a space! (we programmatically parse this to generate the "Contributors" section of the changelog)
```suggestion
  Author: daniel-murray josephberni
```